pkgs/applications/misc/hivemind/default.nix

```diff
@@ -1,9 +1,8 @@
-{ stdenv, buildGoPackage, fetchFromGitHub, runtimeShell }:
+{ stdenv, buildGoModule, fetchFromGitHub, runtimeShell }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "hivemind";
-  version = "1.0.6";
-  goPackagePath = "github.com/DarthSim/hivemind";
+  version = "2020-04-28";
```

Comment:
The package builds and runs with the changes. I was a bit confused by the new version tag at first. I noticed that some packages in this PR have an "unstable" tag in the version or package name.

Reply:
Upstream for this is just pretty bad at releasing packages - 1.0.6 is over a year out of date and no releases have been tagged since then.

Reply:
OK. Thanks for updating the package. LGTM